components/common/FeedbackForm.js

```diff
@@ -69,10 +77,28 @@ const FeedbackForm = ({
   const onSubmit = (formData) => {
     const { name, email, comment, ...additionalResponse } = formData;
 
+    const isCommentEmpty = !comment || comment.trim() === '';
+    const areAllAdditionalFieldsEmpty = Object.values(additionalResponse).every(
+      (value) => {
+        if (Array.isArray(value)) {
+          return value.length === 0;
+        }
+        return value === '' || value === null || value === undefined;
```

Comment:
value.trim() ?

Reply:
Also, you can use `value == null` instead of `value === null || value === undefined`
Also, you should maybe move the null/undefined check first, then you can add the trim()